core/src/main/java/hudson/model/Job.java

```diff
@@ -515,7 +515,7 @@ public void find(String token, List<SearchItem> result) {
             public void suggest(String token, List<SearchItem> result) {
                 find(token, result);
             }
-        }).add("configure", "config", "configure");
```

Comment:
Why?

Reply:
<img width="432" alt="image" src="https://github.com/jenkinsci/jenkins/assets/43062514/b728f9d1-46e2-47fe-bc5d-4801913a1e9f">

Found the UX around these sorts of things a little confusing and offering multiple results for the same thing to be odd. Happy to restore it though.

Reply:
Makes sense to not have multiple items for the same thing (especially if one's just a substring). This removes all of them though?

OTOH There are other interesting actions (or adjacent views) that could make sense to add here, so if there's a larger plan to add more, that would be fine. Or if this just clutters up the search for jobs named "con", that would be pointless.